Browser_Actions/Browser_ActionTests.cs

```diff
@@ -0,0 +1,126 @@
+            elemenPassword.SendKeys("SuperSecretPassword!");
+            var elementLoginButton = Driver.FindElement(By.XPath("//*[@class='radius']"));
+            elementLoginButton.Click();
+            var elementLogoutButton = Driver.FindElement(By.CssSelector("[href*='logout']"));
+            elementLogoutButton.Click();
+            Driver.Navigate().Back();
+            Driver.Navigate().Forward();
+            var pageUrl = Driver.Url;
+            Assert.That(pageUrl, Is.EqualTo($"{MainUrl}/login"));
+            var elementContentLogin = Driver.FindElement(By.Id("content"));
+            Assert.That(elementContentLogin.Text.Contains("Login Page"));
+        }
+
+        [Test]
+        public void NavigateToUrlAndRefresh()
+        {
+            Setup();
+            var elementDynamicLoading = Driver.FindElement(By.CssSelector("[href*='dynamic_loading']"));
+            elementDynamicLoading.Click();
+            var elementExample = Driver.FindElement(By.XPath("//*[@id='content']//a[1]"));
+            elementExample.Click();
+            var elementStartButton = Driver.FindElement(By.TagName("button"));
+            elementStartButton.Click();
+            var wait = new WebDriverWait(Driver, TimeSpan.FromSeconds(20));
+            Assert.That(wait.Until(ExpectedConditions.ElementIsVisible(By.Id("finish"))).Displayed);
+            var currentUrl = Driver.Url;
+            Setup();
+            var homePageUrl = Driver.Url;
+
+            Assert.False(homePageUrl.Equals(currentUrl));
+            Assert.That(homePageUrl, Is.EqualTo($"{MainUrl}/"));
+            Driver.Navigate().Refresh();
+        }
+
+        [Test]
+        public void MaximizeWindowChangeWindowSize()
+        {
+            Setup();
+            var elementLargeDeepDom = Driver.FindElement(By.CssSelector("[href*='large']"));
+            elementLargeDeepDom.Click();
+            var elementLastDom = Driver.FindElement(By.XPath("//*[@id='large-table']/tbody/tr[50]/td[50]"));
+            ((IJavaScriptExecutor)Driver).ExecuteScript("arguments[0].scrollIntoView();", elementLastDom);
+            Assert.That(elementLastDom.Displayed);
+            Driver.Manage().Window.Maximize();
+           
+            var elementFirstLine = Driver.FindElement(By.XPath("//*[@id='content']/div/h3"));
+            ((IJavaScriptExecutor)Driver).ExecuteScript("arguments[0].scrollIntoView();", elementFirstLine);
+            Assert.That(elementFirstLine.Displayed);
+           
+            var newSize = new Size(1000, 800);
+            Driver.Manage().Window.Size = newSize;
+            var currentSize = Driver.Manage().Window.Size;
+            Assert.That(currentSize.Width, Is.EqualTo(newSize.Width));
+            Assert.That(currentSize.Height, Is.EqualTo(newSize.Height));
+        }
+
+        [Test]
+        public void HeadlessMode() 
+        {
+            ChromeOptions options = new ChromeOptions();
```

Comment:
This options are not applied, just written in options instance.
Need to google, how to apply it to our chrome

Reply:
done with your help